server/opts.go

```diff
@@ -593,6 +595,8 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 		o.HTTPPort = int(v.(int64))
 	case "https_port":
 		o.HTTPSPort = int(v.(int64))
+	case "base_path":
```

Comment:
You can have both, we sometimes do that. For instance:
```
case "http_base_path", "base_path":
..
```
to support multiple config names.

Reply:
I'd keep only `http_base_path` as we don't really need to be compatible with anything.

Reply:
Let's go with that and renaming of option, etc.. Will review again when the PR is updated.